src/main/java/org/truffleruby/core/symbol/SymbolTable.java

```diff
@@ -134,7 +134,7 @@ private DynamicObject getDeduplicatedSymbol(Rope rope) {
         }
     }
 
-    private static final int MURMUR_SEED = System.identityHashCode(SymbolTable.class);
+    private static final int CLASS_SALT = 92021474;
```

Comment:
How was this value determined? Does it have any significance? A comment would be helpful here.

Reply:
It’s just a random number so 0.hash doesn’t look like ‘’.hash (hypothetical example). Does ‘salt’ not imply that to you?

Reply:
"class salt" as a phrase seemed odd to me. But I don't think I've ever hard-coded a salt either.

Reply:
It's a salt between classes, not between process runs. We have a separate seed for that, and we have an option to turn it off. Changing them to hard-coded means that our deterministic hashing option actually works.

Reply:
Okay. I'm just suggesting you comment where the value came from, what its utility is, and why it must be hard-coded. Basically just a quick summary of what you've already commented here so the next person to come across it understands as well.